cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtilsTest.java

```diff
@@ -15,6 +15,7 @@
 import java.util.Map;
 
 import org.jetbrains.annotations.NotNull;
+import org.junit.After;
```

Comment:
Could you please migrate this class to JUnit 5 if possible?

Reply:
Yes

Reply:
Awesome, thanks!